src/l3/ip.rs

```diff
@@ -114,7 +97,7 @@ pub struct Ipv6Hdr {
     pub _bitfield_1: BitfieldUnit<[u8; 1usize]>,
     pub flow_lbl: [u8; 3usize],
```

Comment:
`flow_label`

Reply:
but the kernel `ipv6hdr` struct has exactly `flow_lbl` - I would keep it like this
https://elixir.bootlin.com/linux/v2.6.32/source/include/linux/ipv6.h#L117

Reply:
We don't necessarily have to stick to kernel names though. I'm usually for short names, but `lbl` is a bit of exaggeration.